network_modeling/models/cray-network-architecture.yaml

```diff
@@ -385,9 +403,6 @@ network_v2_tds:
         - "lmem"
       shasta_name: "lmem"
       architecture_type: "river_ncn_node_4_port"
-    - lookup_name: ["viz"]
```

Comment:
Is the viz node getting deleted on accident?

Reply:
I dont think I removed it.. Maybe my file source was old and did not have it. Re-added it